benchmarks/src/test/scala/pl/touk/nussknacker/engine/benchmarks/serialization/avro/AvroBenchmark.scala

```diff
@@ -41,7 +38,7 @@ class AvroBenchmark {
       val parsedSchema = ConfluentUtils.convertToAvroSchema(AvroSamples.sampleSchema, Some(1))
       schemaRegistryMockClient.register("foo-value", parsedSchema, 1, AvroSamples.sampleSchemaId)
       val factory: CachedConfluentSchemaRegistryClientFactory =
-        new CachedConfluentSchemaRegistryClientFactory(CacheConfig.defaultMaximumSize, None, None) {
+        new CachedConfluentSchemaRegistryClientFactory(SchemaRegistryCacheConfig.noExpire) {
```

Comment:
I think, maybe we need an entry in changelog? Or also in MigrationGuide, as those are -utils?

Reply:
Good point, added both